auth/domain/src/test/java/com/pronoidsoftware/auth/domain/UserDataValidatorTest.kt

```diff
@@ -0,0 +1,90 @@
+package com.pronoidsoftware.auth.domain
+
+import assertk.assertThat
+import assertk.assertions.isFalse
+import assertk.assertions.isTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class UserDataValidatorTest {
+
+    private lateinit var sut: UserDataValidator
```

Comment:
I'm personally not a friend of such abbreviations that might not be immediately clear to everyone. I understand what you mean here, but if I'd read this name in a random test I probably wouldn't at the first glance. What speaks against just naming this `validator` or so? The class name already says what the subject under test is.